scrapper/src/test/java/edu/java/scrapper/client/StackOverflowWebClientTest.java

```diff
@@ -44,24 +45,25 @@ public void testGettingQuestionInfo() throws IOException {
 
         assertThat(response).isNotNull();
         assertThat(response.items()).isNotEmpty();
-        assertThat(response.items().getFirst().id()).isEqualTo(questionId);
+        assertThat(response.items()).hasSize(1);
     }
 
     @Test
-    public void testResponseOnInvalidQuestionId() throws IOException {
-        File file = ResourceUtils.getFile("classpath:bad-question-id.json");
+    public void testFetchQuestionAnswers() throws IOException {
+        File file = ResourceUtils.getFile("classpath:question-answers-response.json");
         String json = FileUtils.readFileToString(file, StandardCharsets.UTF_8);
 
-        Long badQuestionId = 12345678910L;
-
         wireMockServer
-            .stubFor(get("/questions/%d?site=stackoverflow".formatted(badQuestionId))
+            .stubFor(get(answersPath.formatted(questionId))
                 .willReturn(aResponse()
-                    .withStatus(400)
+                    .withStatus(200)
                     .withBody(json)
                     .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)));
 
-        assertThatExceptionOfType(WebClientResponseException.class)
-            .isThrownBy(() -> stackOverflowWebClient.fetchQuestion(badQuestionId));
+        QuestionResponse response = stackOverflowWebClient.fetchQuestionAnswers(questionId);
+
+        assertThat(response).isNotNull();
+        assertThat(response.items()).isNotEmpty();
```

Comment:
можно подряд вызвать
```suggestion
        assertThat(response.items()).isNotEmpty().hasSize(2);
```